src/hearings/hearings.routes.ts

```diff
@@ -187,6 +188,14 @@ export const ROUTES: Routes = [
           title: 'HMCTS Manage cases | Amend Hearing | Check Answers'
         }
       },
+      {
+        path: 'hearing-actual-add-edit',
+        component: HearingActualAddEditSummaryComponent,
+        canActivate: [HealthCheckGuard],
+        data: {
+          title: 'HMCTS Manage cases | Hearing Actuals | Check details'
+        }
+      },
```

Comment:
The route needs to be put to the actuals/hearing-actual-add-edit instead of request/hearing-actual-add-edit.